lib/cloudformation.py

```diff
@@ -1905,7 +1905,7 @@ def add_lambda(self, key, name, role, file=None, handler=None, s3=None, descript
             subnets (None|list) : List of ids of subnets to grant the lambda access to
             depends_on (None|str|list) : A unique name or list of unique names of resources within the
                                             configuration and is used to determine the launch order of resources
-            runtime (optional[str]) : Lambda runtime to use.  Defaults to "python2.7".
+            runtime (optional[str]) : Lambda runtime to use.  Defaults to "python2.7".  Ignored if handler is not None, but file and s3 are None.
```

Comment:
Should we change the default Python to be 3.7 instead of 2.7?

Reply:
Yes, thought about it, but forgot by the time I pushed. 😛